docs/4.3/metrics_logs_reference.md

```diff
@@ -0,0 +1,99 @@
+# Metrics
+
+## Teleport Prometheus Endpoint
+
+Teleport provides HTTP endpoints for monitoring purposes. They are disabled
+by default, but you can enable them:
+
+```bash
+$ teleport start --diag-addr=127.0.0.1:3000
+```
+
+Now you can see the monitoring information by visiting several endpoints:
+
+* `http://127.0.0.1:3000/metrics` is the list of internal metrics Teleport is
+   tracking. It is compatible with [Prometheus](https://prometheus.io/)
+   collectors.
+
+<!-- Help: Edit this table https://docs.google.com/spreadsheets/d/1lYEGjENdsEoMXGg6LO71-U5Y6PDzPbplut255aacGWo/edit#gid=0 and put the table into https://www.tablesgenerator.com/markdown_tables  -->
```

Comment:
What's this comment about?

Reply:
Creating these tables is a bit of a pain, I think long term we should automate it but didn't want to put the old optimisation horse before the cart.  This makes bulk changes much easier.